src/style.css

```diff
@@ -1,3 +1,71 @@
+html {
+  --blue-color: #00acdc;
+
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.logo {
+  display: flex;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #fff;
+  padding: 0 50px;
+}
+
+.nav {
+  display: inline-block;
+  text-align: center;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin: 0;
+}
+
+.nav__link {
+  display: inline-block;
+  height: 60px;
```

Comment:
You don't actually need both `height` and `line-height` in this case. One of them should be sufficient, just make sure that text is vertically centered.

Reply:
Hi! This is my way to make text centered and I have no idea how to do it another way in this case. Could you help?

Reply:
`line-height` sets the height of text line to 60px. So if there are no additional margins/paddings/borders etc., the parent element would take 60px height. This means that if you remove `height` style in this case, nothing would change.

Reply:
Thanks!